ext/pdo/tests/pdo_031.phpt

```diff
@@ -3,19 +3,17 @@ PDO Common: PDOStatement SPL iterator
 --SKIPIF--
 <?php # vim:ft=php
 if (!extension_loaded('pdo')) die('skip');
-if (!extension_loaded('SPL')) die('skip SPL not available');
 $dir = getenv('REDIR_TEST_DIR');
 if (false == $dir) die('skip no driver');
 require_once $dir . 'pdo_test.inc';
-if (!class_exists('RecursiveArrayIterator', false)) die('skip Class RecursiveArrayIterator missing');
 if (!class_exists('RecursiveTreeIterator', false) && !file_exists(getenv('REDIR_TEST_DIR').'../../spl/examples/recursivetreeiterator.inc')) die('skip Class RecursiveTreeIterator missing');
```

Comment:
`RecursiveTreeIterator` should also always be available. We can also drop the check on L16 then.